hackathon_site/dashboard/frontend/src/components/general/OrderTables/OrderTables.tsx

```diff
@@ -88,3 +102,206 @@ export const GeneralOrderTableTitle = ({
         )}
     </Container>
 );
+
+export const GeneralPendingTable = ({
+    pendingOrder,
+    extraColumn,
+}: {
+    pendingOrder: OrderInTable;
+    extraColumn?: {
+        header: ReactElement;
+        body: (id: number) => ReactElement;
+    };
+}) => {
+    const hardware = useSelector(hardwareSelectors.selectEntities);
```

Comment:
It seems like the tables rendered don't have anything filled in the rows, is it due to this line here (instead of using ```hardware = mockHardware```) ?

Reply:
Yeah good question! So here I integrated redux to get hardware details because this GeneralPendingTable component is used on the participant side too (where redux  is already fully integrated). This line requires you to have hardware in your database (based on the hardware ids in mockPendingOrders). When the table is tested, the redux store is populated with mockHardware.